workshop.md

```diff
@@ -353,57 +381,99 @@ The idea is to tell the model to only generate questions and answers based on th
 ## Generate the flashcards
 
 ```python
+# ----------------- Generate the Flashcards -----------------
```

Comment:
I don't think there's a need to repeat with a comment what the title of the section already states. (This applies to previous code samples)

Reply:
fixed by removing all unnecessary comments